roles/slg.galaxy_stats/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+# Stats scripts directory
+stats_dir: ~/stats_collection
+
+# DB Settings
+db_server: localhost
+database: galaxy
+db_user: galaxy
+db_port: 5432
+db_password: ""
+
+# Galaxy settings
+galaxy_root: /mnt/galaxy/galaxy-app/
```

Comment:
should this be `galaxy_root: /mnt/galaxy`?

Reply:
I think it's referring to the actual code.